client/extensions/woocommerce/app/settings/payments/payment-method-edit.js

```diff
@@ -98,13 +105,12 @@ class PaymentMethodEdit extends Component {
 		const { method, translate } = this.props;
 		const settingsFieldsKeys = method.settings && Object.keys( method.settings );
 		return (
-			<ListItem>
+			<div className="payments__method-edit-pane">
```

Comment:
Where it makes sense, let's use classnames that describe what the element contains. Maybe instead of `payments__method-edit-pane` you could use something like `payments__method-edit-fields` here?